### ISSUE TITLE
0.5 release activities

### ISSUE THREAD

**Issue by SachidanandAlle** [Project-MONAI/MONAILabel#971]
- [x] Upgrade dependency
- [x] Unit Test and Integration pass
- [x] Verify Radiology App for all models (infer, train on single/multi-gpu)
- [x] Verify Pathology App for all models (infer, train on single/multi-gpu)
- [x] Verify Endscopy App for all models (infer, train on single/multi-gpu)
- [x] Verify Bundle App for all models (infer, train on single/multi-gpu)
- [x] Verify client 3D Slicer
- [x] Verify client OHIF (next sample, infer, train)
- [x] Verify client QuPath
- [x] Verify client DSA
- [x] Verify Docker
- [x] Scribbles (OHIF + 3D Slicer) (best works in 3D Slicer due to ROI)
- [x] Verify all documents
- [x] Release candidate

**Comment by SachidanandAlle** [comment]
First step: https://github.com/Project-MONAI/MONAILabel/pull/979

**Comment by SachidanandAlle** [comment]
Depends on: https://github.com/Project-MONAI/MONAILabel/pull/981

**Comment by SachidanandAlle** [comment]
Verified Radiology App for all models (infer, train on single/multi-gpu) - done for segmentation, segmentation_spleen, deepgrow 2d+3d+pipeline(infer), deepedit models

Verified vertebra for infer only @diazandr3s please verify once again.. and may be train if that's applicable.

**Comment by SachidanandAlle** [comment]
Upgrade to rc2: https://github.com/Project-MONAI/MONAILabel/pull/983

**Comment by tangy5** [comment]
Bundle app Verfications:

monai version == 1.0.0rc2

- [ ] Download the monaibundel app and all model-zoo models.
Verification of current Model-Zoo models
- [ ] Train/Inference 1: [spleen_ct_segmentation] 2: [swin_unetr_btcv_segmentation]
- [ ] Inference: 1: [renalStructures_UNEST_segmentation] 2: [wholeBrainSeg_Large_UNEST_segmentation]
- [ ] Train/Inference 1: [brats_mri_segmentation] 2: [mednist_gan] 3:  [spleen_deepedit_annotation]
- [ ] Train/Inference [pancreas_ct_dints_segmentation] [pathology_tumor_detection]
- [ ] Train/Inference[prostate_mri_anatomy] [pathology_tumor_detection]
- [ ] Train/Inference 1: [valve_landmarks] 2: [ventricular_short_axis_3label]

**Comment by tangy5** [comment]
Endoscopy app Verfications:

monai version == 1.0.0rc2

- [ ] Verified CVAT setup, nuclio container deploy.
- [ ] Verified ToolTracking model, inference with CVAT-Nuclio function container, training with MONAI Label. Push MONAI Label fine-tuned model back to Nuclio function container for updating inference model. 
- [ ] Verified Deid for tool section, infer with CVAT-Nuclio function container, training with MONAI Label. 
- [ ] Verified Deepedit for tracking endoscopy images (infer, training).

**Comment by tangy5** [comment]
Clients Verification:
monai version == 1.0.0rc2
monailabel latest 

- [ ] **3D Slicer**
- [ ] **OHIF**
- [ ] **QuPath**
- [ ] **DSA**

**Comment by SachidanandAlle** [comment]
Lets try to update/add items in the main description.. so that things can be checked/tracked easily..

**Comment by diazandr3s** [comment]
> Verified Radiology App for all models (infer, train on single/multi-gpu) - done for segmentation, segmentation_spleen, deepgrow 2d+3d+pipeline(infer), deepedit models
> 
> Verified vertebra for infer only @diazandr3s please verify once again.. and may be train if that's applicable. Also you need to add one section for this here: https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#model-overview (may be you can combine all models under one section..) and mention about pipeline

Please check https://github.com/Project-MONAI/MONAILabel/pull/996

**Comment by tangy5** [comment]
Pathology APP Verification:

Setup QuPath and MONAI Label Pathology app. 
I tested Infer/train, with actual Qupath integration and via integration test. 
There is an issue that cause all run failed. 
All pre-transformation with             **AsChannelFirstd(keys="image")**   are failed.

Looking into the Deepedit_nuclie or other inferers script, the 
Replaced to             **EnsureChannelFirstd(keys="image"),** reported error when calling     **img = MetaTensor(img)**
How is the Metatensor supported currently for pathology images?
